artifacts/anvil-state.json

```diff
@@ -0,0 +1 @@
b9150612e2360208401612d85565b600181811c9082168061327857607f821691505b6020821081036132af577f4e487b71000000000000000000000000000000000000000000000000000000005f52602260045260245ffd5b50919050565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52601160045260245ffd5b818103818111156107c3576107c36132b5565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52603260045260245ffd5b601f821115610a2a57805f5260205f20601f840160051c810160208510156133475750805b601f840160051c820191505b81811015611f6d575f8155600101613353565b67ffffffffffffffff83111561337e5761337e6130c9565b6133928361338c8354613264565b83613322565b5f601f8411600181146133c3575f85156133ac5750838201355b5f19600387901b1c1916600186901b178355611f6d565b5f83815260208120601f198716915b828110156133f257868501358255602094850194600190920191016133d2565b508682101561340e575f1960f88860031b161c19848701351681555b505060018560011b0183555050505050565b81835281816020850137505f602082840101525f6020601f19601f840116840101905092915050565b602081525f612c9d602083018486613420565b5f63ffffffff821663ffffffff8103613477576134776132b5565b60010192915050565b63ffffffff8181168382160290811690818114612973576129736132b5565b815167ffffffffffffffff8111156134b9576134b96130c9565b6134cd816134c78454613264565b84613322565b6020601f8211600181146134ff575f83156134e85750848201515b5f19600385901b1c1916600184901b178455611f6d565b5f84815260208120601f198516915b8281101561352e578785015182556020948501946001909201910161350e565b508482101561354b57868401515f19600387901b60f8161c191681555b50505050600190811b01905550565b606081525f61356d606083018789613420565b8281036020840152613580818688613420565b9150508260408301529695505050505050565b63ffffffff81811683821601908111156107c3576107c36132b5565b5f81518060208401855e5f93019283525090919050565b5f612c9d6135d483866135af565b846135af565b65ffffffffffff81811683821601908111156107c3576107c36132b5565b6001600160a01b03851681526001600160a01b0384166020820152826040820152608060608201525f61362e6080830184612d2e565b9695505050505050565b5f60208284031215613648575f5ffd5b81516114ba81612ce6565b65ffffffffffff82811682821603908111156107c3576107c36132b5565b7f4e487b71000000000000000000000000000000000000000000000000000000005f52603160045260245ffdfea26469706673582212208dc50a9ac33e1bc3e64d58d538d08434023bceb730614b89d18cca265e3ec93864736f6c634300081c0033","nonce":2,"gas_used":3278883},"receipt":{"type":"0x2","status":"0x1","cumulativeGasUsed":"0x320823","logs":[{"address":"0x9fe46736679d2d9a65f0992f2272de9f3c7fa6e0","topics":["0x2f8788117e7eff1d82e926ec794901d17c78024a50270940304540a733656f0d","0x0000000000000000000000000000000000000000000000000000000000000000","0x000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfffb92266","0x000000000000000000000000f39fd6e51aad88f6f4ce6ab8827279cfffb92266"],"data":"0x"},{"address":"0x9fe46736679d2d9a65f0992f2272de9f3c7fa6e0","topics":["0xbd79b86ffe0ab8e8776151514217cd7cacd52c909f66475c3af44e129f0b00ff","0xa49807205ce4d355092ef5a8a18f56e8913cf4a201fbe287825b095693c21775","0x0000000000000000000000000000
```

Comment:
do you want this file in Version control?

Reply:
I've seen worse, but yes, I agree it's not the most optimal. It does the job on having an anvil state though.